cuda/sample_consensus/src/multi_ransac.cu

```diff
@@ -208,8 +208,8 @@ namespace pcl
               // Compute the k parameter (k=std::log(z)/std::log(1-w^n))
               float w = (float)((float)min_nr_in_shape / (float)nr_remaining_points);
               float p_no_outliers = 1.0f - pow (w, 1.0f);
-              p_no_outliers = (std::max) (std::numeric_limits<float>::epsilon (), p_no_outliers);       // Avoid division by -Inf
-              p_no_outliers = (std::min) (1.0f - std::numeric_limits<float>::epsilon (), p_no_outliers);   // Avoid division by 0.
+              p_no_outliers = (max) (std::numeric_limits<float>::epsilon (), p_no_outliers);       // Avoid division by -Inf
+              p_no_outliers = (min) (1.0f - std::numeric_limits<float>::epsilon (), p_no_outliers);   // Avoid division by 0.
```

Comment:
awkward parenthesis

Reply:
@kunaltyagi said not to change formatting if it passed CI, but I can change these if you approve

Reply:
I'll let him decide. Just pointing out.

Reply:
I don't like them either, modified.